tests/unittest/_torch/attention/sparse/test_cute_dsl_gvr_topk_decode.py

```
@@ -234,7 +305,213 @@ def test_cute_dsl_gvr_topk_decode(
+    ids=["all_short", "1/4_long", "half_long", "3/4_long", "all_long"],
+)
+def test_lb_prepare_partition(B, ratio):
+    """Prepare kernel: counters + order_row match a numpy partition reference.
+
+    Builds synthetic seq_lens with a controlled long/short ratio, shuffles
+    them, then verifies the kernel partitions the request_ids into
+    [long...][short...] correctly.
+    """
+    long_threshold = 64 * 1024
+    torch.manual_seed(B * 1000 + int(ratio * 100))
+    n_long_expect = int(round(B * ratio))
```

Ruff (0.15.15)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)